localstripe/resources.py

```diff
@@ -1858,7 +1861,7 @@ def status(self):
             return 'processing'
 
     @property
-    def charges(self):
+    def charges(self):  # deprecated
```

Comment:
Not related to this commit but I took advantage of it to add this small comment.